app/src/main/res/layout/activity_movie_details_sec.xml

```diff
@@ -0,0 +1,340 @@
+        android:background="@color/backgroundColor">
+
+
+
+
+
+    <ImageView
+        android:id="@+id/img_back"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="30dp"
+        android:src="@drawable/orig"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+    </ImageView>
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@drawable/gradient">
+
+    </View>
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintWidth_percent="0.25"
+        android:layout_marginBottom="160dp"
+        android:drawableStart="@drawable/ic_path"
+        android:drawablePadding="8dp"
+        android:text="Back"
+        android:textColor="@color/forBack"
+        android:textSize="12dp"
+
+        app:layout_constraintBottom_toBottomOf="@id/img_back"
+        app:layout_constraintStart_toStartOf="@id/img_back"
+        app:layout_constraintTop_toTopOf="@id/img_back">
+
+    </TextView>
+
+    <TextView
+        android:id="@+id/text_13"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="16dp"
+        android:text="13+"
+        android:textColor="@color/white"
+        android:textSize="12dp"
+        app:layout_constraintEnd_toStartOf="@id/img_back"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/img_back">
+
+    </TextView>
+
+    <TextView
+        android:id="@+id/name"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:gravity="left"
+
+        android:text="Avengers: End Game"
+        android:textColor="@color/textCol"
+        android:textSize="40dp"
+        app:layout_constraintLeft_toLeftOf="@id/text_13"
+        app:layout_constraintRight_toLeftOf="@id/text_13"
+        app:layout_constraintStart_toStartOf="@id/text_13"
+        app:layout_constraintTop_toBottomOf="@id/text_13"
+
+        app:layout_constraintWidth_percent="0.6">
+
+    </TextView>
+
+    <TextView
+        android:id="@+id/action"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="8dp"
+        android:text="Action, Adventure, Fantasy"
+        android:textColor="@color/actionCol"
+        android:textSize="14dp"
+        app:layout_constraintStart_toStartOf="@id/name"
+        app:layout_constraintTop_toBottomOf="@id/name"
+
+        >
+
+    </TextView>
+
+    <ImageView
+        android:id="@+id/firstStar"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingRight="2dp"
+
+        android:src="@drawable/star"
```

Comment:
app:layout_constraintBottom_toBottomOf="@id/reviews"
app:layout_constraintEnd_toStartOf="@id/secondStar"
remove here and in stars below.